lacework-agent/templates/configmap.yaml

```diff
@@ -36,13 +36,6 @@ data:
        },
        {{- end }}
      {{- end }}
-     {{- with .Values.laceworkConfig.codeaware }}
-       {{- if .enable }}
-        "codeaware": {
-          "enable": "all"
```

Comment:
just double checking here, we will disable CAA on by default in this fleet-wide release? @geremy-condra

Reply:
No, this is likely because the CAA changes didn't make it to the release branch.

Reply:
got it, good catch. Will remove